### DISCUSSION TITLE
filtering for asynchronous channel

### DISCUSSION THREAD

**Discussion by timheistek** [AllenInstitute/MIES#1480] [Q&A] [answered]
Hi Thomas

We would like to use  asynchronous data acquisition for monitoring the temperature. The signal from our temperature controller has some high frequency noise (~1kHz) so the alarm function doesn't work properly. Do I understand correctly that the alarm state is determined by sampling single points from the asynchronous channel? Would it be possible to use a low pass filter on the asynchronous channel in MIES and use this for determining the alarm state?

**Comment by t-b** [comment, marked as answer]
>  Do I understand correctly that the alarm state is determined by sampling single points from the asynchronous channel?

Yes, we do only do a single read.

> Would it be possible to use a low pass filter on the asynchronous channel in MIES and use this for determining the alarm state?

Technically possible, but it would require us to sample a lot more points so that we can filter the data properly.

A totally different idea: Have you tried to figure out the source of the HF noise? From my understanding you should also be able to filter out the noise on the electronics level with a capacitor in the line.
